packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.spec.ts

```diff
@@ -166,5 +166,23 @@ describe("XYChartComponent", () => {
             });
             expect(spy).toHaveBeenCalledWith({ payload: timeframe });
         });
+
+        it("should return true if the series is set to interactive", () => {
+            component.configuration = { interaction: "series" } as ITimeseriesWidgetConfig;
+            component.ngOnChanges(initializationChanges);
+            expect(component.isSeriesInteractive(component.chartAssist.legendSeriesSet[0])).toBe(true);
```

Comment:
We may need tests for `link` and `secondaryLink` as well since if those are truthy, this method should return. Additionally, if they're all false, this method should return false.